sources/b12/sections/02-data-collection.adoc

```diff
@@ -15,10 +16,11 @@ Single beam echo-sounders collect a single depth measurement from a relatively n
+Multibeam echo-sounders collect depth measurements by forming many receive beams in a wide arc below (or in the case of forward-looking navigation sonar, in front of) the vessel. Multibeam echo- sounders provide a much more detailed representation of the seafloor than single beam depth sounders, and thus can provide additional information about hazards or objects on the seafloor. Multibeam echo- sounders are often found on research vessels, as well as some commercial vessels, expedition cruise ships, and recreational vessels. Vessels equipped with multibeam echo-sounders that wish to contribute data to the DCDB’s established multibeam pipeline should contact the DCDB directly at mailto:mb.info@noaa.gov[].
 
 ==== Positioning Systems
-Positioning systems help mariners determine their location on the Earth’s surface and provide vital information for CSB. Without accurate location information, CSB has little value. Most vessels carry a Global Navigation Satellite System (GNSS), such as GPS, GLONASS or any other constellation, which obtain position fixes automatically. GNSS positions are typically provided once per second and are accompanied by a time and date stamp. CSB data collection systems should provide a position and timestamp with every depth reading. This allows data users to accurately position depth measurements and apply corrections to the data if needed. The GNSS can also output information about the quality of the signal and interruptions in service, and these data should also be logged, if possible.
+
+Positioning systems help mariners determine their location on the Earth’s surface and provide vital information for CSB. Without accurate location information, CSB has little value. Most vessels carry a <<term-gnss>>, such as GPS, GLONASS or any other constellation, which obtain position fixes automatically. GNSS positions are typically provided once per second and are accompanied by a time and date stamp. CSB data collection systems should provide a position and timestamp with every depth reading. This allows data users to accurately position depth measurements and apply corrections to the data if needed. The GNSS can also output information about the quality of the signal and interruptions in service, and these data should also be logged, if possible.
```

Comment:
As above. You might need `<<term-gnss,GNSS>>`

Reply:
Done